alpha/lib/model/entry.php

```diff
@@ -363,11 +363,14 @@ public function setStatusReady ( $force = false )
 	 */
 	public function setModerationStatus($v)
 	{
-		if($v == $this->getModerationStatus())
-			return $this;
 			
-		if($v == entry::ENTRY_MODERATION_STATUS_PENDING_MODERATION || $v == entry::ENTRY_MODERATION_STATUS_FLAGGED_FOR_REVIEW)
+		$moderationStatuses = array(entry::ENTRY_MODERATION_STATUS_PENDING_MODERATION, $v == entry::ENTRY_MODERATION_STATUS_FLAGGED_FOR_REVIEW);
```

Comment:
this resolves to 0/1, not to the enum. looks like copy error
$v == entry::ENTRY_MODERATION_STATUS_FLAGGED_FOR_REVIEW